Add installation docs and learning path structure (#3)
anz-bank/sysl-website: docs/examples-dev.md

sidebar_label: Developer Use Cases
---

Doc Reference: 
* ./examples/gen/codegen
* api-catalog
* sysl-go
* sysl-template

## As a Developer
As a developer, you have worked across the software delivery life cycle from design all the way to support. Your goal is to support your squad by building and utilising vendor tools to create solutions to meet your team’s mission.

> You want to be efficient and gets frustrated when you are slowed down.

Reviewer: ```suggestion
You want to be efficient and get frustrated when you are slowed down.
```

You care about the quality of code and likes to be known as an expert.
You believe documentation is an important resource to learn and seek help.

> You like to use digital tools.

Reviewer: Make these bullets to keep them in separate lines.

> How can Sysl help you: Sysl can help increase you productivity by eliminating the tedious manual time to code and reliably generating codes from a single source of truth.

Reviewer: ```suggestion
How can Sysl help you: Sysl can help increase you productivity by eliminating tedious manual coding effort. Sysl can reliably generate boilerplate code from a single source of truth.
```

> How can Sysl help you: Sysl can help increase you productivity by eliminating the tedious manual time to code and reliably generating codes from a single source of truth.

Reviewer 2: I like the vibes of this a lot but I think we spend a little but too long on WHO the developer is, and not enough time on the benefits and specific problems that sysl can solve. I think something like this might be more targeted:

> As a developer, you've gotten over the initial excitement of building REST applications and you find your job becoming repetitive, every task seems to be to expose this endpoint, or wire this frontend application to this service. Your time is spent between writing boilerplate code and constantly trying to keep documentation up to date. What if it didn't have to be this way?
> 
> Sysl can let you reclaim your time by generating your boilerplate code, as well as your documentation so that you can focus on what you like doing most: building awesome applications!

## Get Started

> 

Reviewer 2: Love the breakdown into each scenario, makes it really easy to find your exact use case. I wonder if each of these scenarios should be in a separate page in the future though.

Also, I think a table of contents that lists each scenario would be really useful, so that users can just jump to the scenario they need

Author: There are table of contents on the right side of the page so each scenario title would be clearly showed.
